python/test/pyRoot_TEST.py

```diff
@@ -0,0 +1,262 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from ignition.math import Vector3d, Pose3d
+from sdformat import Error, Root, SDF_VERSION, SDF_PROTOCOL_VERSION, World
```

Comment:
I believe `SDF_VERSION` and `SDF_PROTOCOL_VERSION` are not yet defined

Reply:
Done https://github.com/ignitionrobotics/sdformat/pull/967/commits/6d58711b2c3eb125fe7034ae179a359853a36958